### PR TITLE
fix new triggerlog recapture on UPDATE

### PR DESCRIPTION
**Pull request by syphar**
when recapturing an UPDATE, we should keep the columns in the record.

### CODE REVIEW

**Review by syphar — COMMENTED** [review]
- `.values` and the other HStore-Fields in the triggerlog are actually test-fields in the database. how to handle?

**Comment by amureki** [comment]
For the reference: https://devcenter.heroku.com/articles/writing-data-to-salesforce-with-heroku-connect#structure

File: heroku_connect/models.py [hunk 1]
```diff
@@ -102,9 +101,8 @@ class TriggerLogAbstract(models.Model):
                              choices=TRIGGER_LOG_STATE_CHOICES)
     sf_message = models.TextField(editable=False, null=True, blank=True)
 
-    # HStoreFields need 'django.contrib.postgres' in INSTALLED_APPS and hstore postgres extension
-    values = HStoreField(editable=False, null=True, blank=True)
-    old = HStoreField(editable=False, null=True, blank=True)
+    values = models.TextField(editable=False, null=True, blank=True)
+    old = models.TextField(editable=False, null=True, blank=True)
```
Comment (amureki) [inline heroku_connect/models.py:105]: Does Django generate a migration for such case?
So, for example if we introduce this change to systems already using old code, would that work without an issue?

**Review by amureki — COMMENTED** [review]
Small comment to clarify thing I am not sure without checking this by myself...

File: heroku_connect/models.py (hunk 1 above)
Comment (syphar) [reply heroku_connect/models.py:105]: it's a breaking change on the library level

Comment (syphar) [reply heroku_connect/models.py:105]: (but no migrations, since it is an unmanaged model)

Comment (amureki) [reply heroku_connect/models.py:105]: Ah, true, right!
Then we would need to cover this in the release notes.

**Review by amureki — APPROVED** [review]
👍

**Pull request merged**